tests/integrations/celery/test_celery.py

```diff
@@ -278,42 +277,6 @@ def dummy_task(x, y):
     assert not events
 
 
-def test_broken_prerun(init_celery, connect_signal):
```

Comment:
please do not delete tests but skip them with pytest and a reason

Reply:
ok. done.

Reply:
I think these are fine to delete if their only purpose is to test old celery versions we've dropped support for. No use keeping them around.